pkg/crc/machine/vsock.go

```diff
@@ -146,3 +146,21 @@ func getSSHTunnelURI() string {
 	}
 	return u.String()
 }
+
+func portFallbackWarning(ingressHTTPPort, ingressHTTPSPort uint) string {
+	var fallbackPortWarning string
+	userAssignedHTTPPort := strconv.FormatUint(uint64(ingressHTTPPort), 10)
+	userAssignedHTTPSPort := strconv.FormatUint(uint64(ingressHTTPSPort), 10)
+	fallbackPortWarningTmpl := `
+Note: Port %s is Assigned.
+You have to add this port to the application route to access
+it. like %s://myapp.apps-crc.testing:%s
```

Comment:
`to acces it. like %s ..` -> `to access it, such as %s://myapp.apps-crc.testing:%s`